### DISCUSSION TITLE
UI not loading traces from zipkin datasource when queried by trace selector

### DISCUSSION THREAD

**Discussion by p1742000** [grafana/grafana#70485] [General]
Hello team,
We used zipkin datasource for our backend. but we are not able to see traces when querying using trace selector. it is giving this error : 

"Failed to load spans from zipkin, Cannot read properties of undefined (reading 'name')"

<img width="1182" alt="MicrosoftTeams-image (6)" src="https://github.com/grafana/grafana/assets/61838673/e9c27c67-512c-43e2-bd19-972b206f0cd6">

but we are getting traces while querying through search by traceID. we have also tried with latest version of zipkin lens UI, we are successfully getting all traces. So, is the zipkin datasource plugin needs to be upgraded to the latest version ? 

<img width="1233" alt="MicrosoftTeams-image (8)" src="https://github.com/grafana/grafana/assets/61838673/75abfd86-5bfa-45d8-8f2e-487de48b9144">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
